### DISCUSSION TITLE
is there a way to modify the new version value when using a regex manager ?

### DISCUSSION THREAD

**Discussion by Gabriel-Ladzaretti** [renovatebot/renovate#25558] [Request Help] [answered]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us what version of Renovate you run.

latest

### If you're self-hosting Renovate, select which platform you are using.

github.com

### Was this something which used to work for you, and then stopped?

is there a way to drop the below v prefix ?

Current version value
```dockerfile
# renovate: datasource=node depName=node
ENV DOCKERFILE_NODE_VERSION=18.18.0
```

Actual new version value
```dockerfile
# renovate: datasource=node depName=node
ENV DOCKERFILE_NODE_VERSION=v20.9.0
```

### Wanted end result.

new version value
```dockerfile
# renovate: datasource=node depName=node
ENV DOCKERFILE_NODE_VERSION=20.9.0 <- no `v` prefix
```

### What you tried so far.

The following regex manager

```json
"regexManagers": [
    {
      "description": "Update _VERSION variables in Dockerfiles",
      "fileMatch": ["(^|/|\\.)Dockerfile$", "(^|/)Dockerfile\\.[^/]*$"],
      "matchStrings": [
        "# renovate: datasource=(?<datasource>[a-z-]+?) depName=(?<depName>.+?)(?: (?:packageName|lookupName)=(?<packageName>.+?))?(?: versioning=(?<versioning>[a-z-]+?))?\\s(?:ENV|ARG) .+?_VERSION=(?<currentValue>.+?)\\s"
      ],
      "versioningTemplate": "{{#if versioning}}{{versioning}}{{else}}semver{{/if}}"
    }
  ]
```

**Comment by secustor** [comment, marked as answer]
Look into [extractVersion](https://docs.renovatebot.com/configuration-options/#extractversion) there is an example how to remove leading a leading `v`
